Bixal/move.mil
# Feature/158296845/fix video embeds

Initial solution to [lack of video embed responsiveness](https://www.pivotaltracker.com/story/show/158296845).....note that it will also require a markup change to apply these styles to the existing video on the [Air Force page](https://move.mil/service-specific-information/air-force).

We may want to consider adding a dedicated field, but this is a good start that should at least take care of the existing video and allow future videos to be responsive with proper client training.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Adds a new WYSIWYG style option for a wrapper div with class `video-embed`.
- Adds css to ensure iframe embeds within that wrapper behave responsively, even if height and width attributes are included.

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Import config and build theme assets
1. Go to edit existing video on `/service-specific-information/air-force` page
1. Change WYSIWYG box with the video (about halfway down the page) to `Full HTML` if it isn't that already.
1. Go to the WYSIWYG styles dropdown and select the `Video Embed` option.
1. Click on the `<source>` button to view source code
1. Cut/copy the existing `<iframe>` tag and paste it within the new div with class `video-embed`.
1. Save the changes to view results.

## Screenshots

### Live (unfixed):
<img width="711" alt="responsive video embed - live unfixed" src="https://user-images.githubusercontent.com/12779671/44678740-16dee700-aa07-11e8-9f1e-6c5269812ecd.png">

<img width="402" alt="responsive video embed - live unfixed mobile" src="https://user-images.githubusercontent.com/12779671/44678756-1d6d5e80-aa07-11e8-824a-b9ad2fe14413.png">


### Local (fixed):
<img width="607" alt="responsive video embed - local fixed" src="https://user-images.githubusercontent.com/12779671/44678772-2c541100-aa07-11e8-99e6-144ec8092719.png">

<img width="495" alt="responsive video embed - local fixed mobile" src="https://user-images.githubusercontent.com/12779671/44678794-3249f200-aa07-11e8-961b-bd20526a1240.png">
